firefox-ios/Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -115,17 +129,39 @@ class Setting: NSObject {
 
     init(
         title: NSAttributedString? = nil,
+        centeredTitle: NSAttributedString? = nil,
         footerTitle: NSAttributedString? = nil,
         cellHeight: CGFloat? = nil,
         delegate: SettingsDelegate? = nil,
         enabled: Bool? = nil
     ) {
         self._title = title
+        self._centeredTitle = centeredTitle
         self._footerTitle = footerTitle
         self._cellHeight = cellHeight
         self.delegate = delegate
         self.enabled = enabled ?? true
     }
+
+    private func setupCenteredLabelConstraints(contentView: UIView) {
```

Comment:
If we have a custom cell for the centered style we won't need this code anymore as it would live in the cell.